addons/website/data/ir_asset.xml

```diff
@@ -25,7 +25,8 @@
         </record>
 
         <record id="website.configurator_tour" model="ir.asset">
-            <field name="name">website.configurator_tour</field>
+            <field name="key">website.configurator_tour</field>
```

Comment:
Is not the key equal to the xml_id per default? Or did they not implement that for ir.asset? :/

Reply:
> Is not the key equal to the xml_id per default?

It doesn't seem to be the case:

![Screenshot from 2021-06-01 16-11-03](https://user-images.githubusercontent.com/49784121/120337863-fbaa1000-c2f3-11eb-8234-833f047f6f42.png)

Reply:
Normal @Arcasias ?

Reply:
> Or did they not implement that for ir.asset?

No we did not, although this might be a good idea.

> Normal @Arcasias ?

Yes, as you mentioned the key is set on `theme.ir.asset` records to identify them and override them in specific website. The name is just an arbitrary string used to identify assets in the UI.

Reply:
@Arcasias 

> Yes, as you mentioned the key is set on theme.ir.asset records to identify them and override them in specific website

I am not sure what you mean :/ theme.ir.asset are templates to create ir.asset, right? So you define a key on the theme.ir.asset, that creates an ir.asset with that key... but then how is it relevant if the asset they may override do not have a key themselves? I am lost :grimacing:

Reply:
@qsm-odoo Well this may be a regression compared to the previous system, but now only `ir.asset` created by a theme can be overridden, so that's why they are the only one having a key. This didn't look like an issue as we have converted (supposedly) all overridable assets to `theme.ir.asset` records during the refactoring.

Reply:
> but now only ir.asset created by a theme can be overridden [...] we have converted (supposedly) all overridable assets to theme.ir.asset

I don't understand this: all ir.asset created from a theme.ir.asset are all website-specific by definition, right? (since they apply for one website). So what overridable assets in themes are we talking about? :thinking: I think we debated that this was a potential feature with assets views and that we should keep it but that there were nothing to convert anymore, right? But since you implemented a key system (which is good), I wonder where you use it. And do we agree that if all ir.asset had their "key" field set to the xmlid by default (like ir.attachment I think?), then the feature to "override" an asset would work outside of themes?

---

Unrelated: playing with those I found another bug: install a theme, change a theme.ir.asset (change a "before" to an "after" for example -> which is what I did to solve a conversion to ir.asset bug) then do a -u of the theme -> the ir.asset is not updated. Should I report that in a task for your team? Kinda a big bug to not be able to update themes :wink:

Reply:
> So what overridable assets in themes are we talking about?

All `ir.asset` records created by `theme.ir.asset` records.

> if all ir.asset had their "key" field set to the xmlid by default (like ir.attachment I think?), then the feature to "override" an asset would work outside of themes?

In theory it should work, yes. Although this would need some testing

> Should I report that in a task for your team?

Yup, that one is definitely not intended 😅